releasenotes/notes/trace_agent_exit_gracefully_no_api_key-aaa2ad5a46506604.yaml

```diff
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    APM: The Trace Agent exits gracefully when the API key is not specified.
```

Comment:
Maybe better
```suggestion
    APM: The agent now exits with code 0 when the API key is not specified. This is so to prevent the Windows SCM 
    from restarting the process. 
```